src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -430,6 +437,40 @@ void DynamoViewModelRequestAboutWindow(DynamoViewModel model)
             aboutWindow.ShowDialog();
         }
 
+        private void OutsideGalleryView_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        {
+            dynamoViewModel.CloseGalleryCommand.Execute(null);
+            e.Handled = true;
+        }
+
+        void DynamoViewModelRequestShowGallery()
+        {
+            if (galleryView == null) //On-demand instantiation
+            {
+                galleryView = new GalleryView(new GalleryViewModel(dynamoViewModel));
+                Grid.SetColumnSpan(galleryUi, mainGrid.ColumnDefinitions.Count);
+                Grid.SetRowSpan(galleryUi, mainGrid.RowDefinitions.Count);
```

Comment:
We don't need to specify `ColumnSpan` and `RowSpan` because the owning grid `galleryUi` has only one column and one row.